ingestion/src/metadata/cmd.py

```diff
@@ -123,4 +131,80 @@ def report(config: str) -> None:
         ) from exc
 
 
+@metadata.command()
+@click.option(
+    "-t",
+    "--type",
+    help="Workflow config",
+    required=True,
+)
+@click.option(
+    "-p",
+    "--path",
+    help="Workflow config",
+    required=False,
+)
+def docker_run(type: str, path: str = "") -> None:
+    """
+    Checks Docker Memory Allocation
+    Run Local Docker - metadata docker-run -t local
+    Run Latest Release Docker - metadata docker-run -t latest
+    """
+    try:
+        client = docker.from_env()
+        docker_info = client.info()
+        calc_gb = 1024 * 1024 * 1000
+        if docker_info["MemTotal"] < (3 * 1024 * 1024 * 1000):
+            raise MemoryError(
+                f"Please Allocate More memory to Docker.\nRecommended: 4GB\nCurrent: "
+                f"{round(float(docker_info['MemTotal']) / calc_gb, 2)}"
+            )
+        if type == "local":
+            logger.info("Running Local Docker")
+            if path == "":
+                raise ValueError("Please Provide Path to local docker-compose.yml file")
+            else:
+                subprocess.run(
+                    f"docker compose -f {path} up --build -d",
+                    shell=True,
+                )
+        elif type == "latest":
+            logger.info("Running Latest Release Docker")
```

Comment:
can we add start time and end time before and after the docker-compose and log how long its taking to start the containers

Reply:
made those change in the recent commits.